lib/vanagon/component/dsl.rb

```diff
@@ -173,6 +173,12 @@ def install_service(service_file, default_file = nil, service_name = @component.
           @component.service = OpenStruct.new(:name => service_name, :service_command => File.read(service_file).chomp)
           # Return here because there is no file to install, just a string read in
           return
+        when "windows"
+          puts "Now we know how to install windows - sort of - eventually"
```

Comment:
We can also always just return an empty sting here since we haven't implemented this yet